### DISCUSSION TITLE
bookmarklets

### DISCUSSION THREAD

**Discussion by 3s7vn** [3kh0/ext-remover#626] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
very bad news, i think the school disabled javascript, everytime i run something with javascript: , it doesn't work, what do I do?

**Comment by piplup7575** [comment, marked as answer]
just use ltmeat

**Reply by thebossjeff** [reply]
sadly, this does not work for all of us.

**Reply by 3s7vn** [reply]
unfortunately, LTMEAT doesnt work either, my school blocklisted chrome://hang and chrome://kill

**Reply by Aka-but-nice** [reply]
Haha, I see that they have been aware of this exploit for a while now. The best thing to do would be to use the incognito exploit which joining [Titanium Network](https://discord.gg/unblock) will tell you more about it & how it functions as well as how to perform it.

**Reply by AshtonDavies** [reply]
Everyone who has LTMEAT blocked, follow my workaround without hang and kill commands: https://github.com/3kh0/ext-remover/discussions/671#discussioncomment-6105092
